PubHub/PubHub.API/Domain/Identity/Account.cs

```diff
@@ -63,9 +63,10 @@ public sealed class Account : IdentityUser<int>
             }
         }
         public required DateTime LastSignIn { get; set; }
+        public bool IsDeleted { get; set; }
 
         #region Navs
-        public required AccountType AccountType { get; set; }
+        public AccountType? AccountType { get; set; }
```

Comment:
Why was this made optional? Shouldn't we always indicate the account type?

Reply:
Because it was my understanding that we use eager loading? (: